src/components/RecipeList.js

```diff
@@ -0,0 +1,26 @@
+require('normalize.css/normalize.css');
+require('styles/App.css');
+
+import React from 'react';
+
+var RecipeList = React.createClass({
+  render: function() {
```

Comment:
Yes. `LogoComponent` uses the `class ... extends` syntax. On the other hand the official tutorials were using `createClass` factory. Is there a rule-of-thumb to pick one or another?